ansible/group_vars/all/vars

```diff
@@ -0,0 +1,23 @@
+# Default test env templates
+testenv_testbed_vmware_template: Template-testbed-201803050718
+testenv_controller_vmware_template: Template-CentOS-7.4
+
+# Testenv block definition
+provisioning:
```

Comment:
`provisioning` seems a little bit ambiguous, perhaps use the comment as variable name?

Reply:
`testenv_block`?

Reply:
Sgtm